scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -71,9 +71,13 @@ EMBED_PROVIDERS, $scope, editor, config, $injector) {
                         name: EMBED_PROVIDERS.youtube
                     }
                 ];
-                // prepend with custom handlers from config
-                if (config.editorEmbedCodeParsers) {
-                    knownProviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownProviders);
```

Comment:
:+1: thanks Edouard

On Tue, Aug 2, 2016 at 11:36 AM, Edouard <notifications@github.com> wrote:

> In scripts/superdesk/editor2/add-embed.ctrl.js
> <https://github.com/superdesk/superdesk-client-core/pull/768#discussion_r73132133>
> :
>
> > @@ -71,9 +71,13 @@ EMBED_PROVIDERS, $scope, editor, config, $injector) {
> >                          name: EMBED_PROVIDERS.youtube
> >                      }
> >                  ];
> > -                // prepend with custom handlers from config
> > -                if (config.editorEmbedCodeParsers) {
> > -                    knownProviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownProviders);
>
> I just merged superdesk/superdesk-core#508
> <https://github.com/superdesk/superdesk-core/pull/508>, so you can now
> add apps.vidible to the installed app in this branch
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/superdesk/superdesk-client-core/pull/768/files/c2e87c1f948aecd677de1044d5df5ca091671dba#r73132133>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGYGlN2xXA0RJZGJ6V10-CYqi35JBxOFks5qbx24gaJpZM4JWCm7>
> .
>